types/qtypings-nodejs-global/typings/fn.d.ts

```diff
@@ -0,0 +1,35 @@
+/** 任意回调参数的回调函数 */
+interface QFnAnyArgs{
+    (... args:any[]):void;
+}
+
+/** 无回调参数的回调函数 */
+interface QFnEmptyArgs{
+    ():void;
+}
+
+/** 任意回调参数的回调函数, 返回类型T */
+interface QFnAnyArgsReturnT<T>{
+    (... args:any[]): T;
+}
+
+/** 是否成功回调函数 */
+interface QFnIsSuccess{
+    (isSuccess: boolean):void;
+}
+
+/** 是否成功回调函数, 第2个参数开始任意回调参数*/
+interface QFnIsSuccessAnyArgs{
+    (isSuccess: boolean, ...args: any[]):void;
+}
+
+/** 错误则无result，有result则无错误 */
+interface QFnErrorOrResult{
+    // eslint-disable-next-line @definitelytyped/no-any-union
+    (err: Error | any | undefined, result?: any, ...args: any[]): void;
```

Comment:
This signature will have result with parameter `err` typed as `any` so please fix that and remove the eslint disable line.

Reply:
bug fixed